### PR TITLE
Fixes #14418: Enhance rudder-pkg to allow user to install plugin from repo

### PR DESCRIPTION
**Pull request by Fdall**
https://issues.rudder.io/issues/14418

### CODE REVIEW

File: rudder-server-relay/SOURCES/rudder-pkg [hunk 1]
```diff
@@ -1,30 +1,45 @@
-#!/usr/bin/python
+#!/usr/bin/python2.7
```
Comment (amousset) [inline rudder-server-relay/SOURCES/rudder-pkg]: We should be compatible with python3

File: rudder-server-relay/SOURCES/rudder-pkg-files/rudderPkgUtils.py [hunk 2]
```diff
@@ -0,0 +1,326 @@
...
+  db_save()
+
+############# Variables ############# 
+""" Defining global variables."""
+
+LOG_PATH = "/var/log/rudder/rudder-pkg.log"
+CONFIG_PATH = "/opt/rudder/etc/rudder-pkg.conf"
+FOLDER_PATH = "/var/rudder/tmp/plugins"
+INDEX_PATH = FOLDER_PATH + "/rpkg.index"
+GPG_HOME = "/tmp/toto"
```
Comment (amousset) [inline rudder-server-relay/SOURCES/rudder-pkg-files/rudderPkgUtils.py]: I guess not

File: rudder-server-relay/SOURCES/rudder-pkg-files/rudderPkgUtils.py [hunk 3]
```diff
@@ -0,0 +1,326 @@
...
+GPG_RUDDER_KEY = "/var/rudder/rudder_apt_key.pub"
```
Comment (amousset) [inline rudder-server-relay/SOURCES/rudder-pkg-files/rudderPkgUtils.py]: We need to find a better place for this, probably somewhere in `/opt/rudder` as this is part of the package and does not change.

File: rudder-server-relay/SOURCES/rudder-pkg-files/rudderPkgUtils.py [hunk 4]
```diff
@@ -0,0 +1,326 @@
...
+GPG_RUDDER_KEY_FINGERPRINT = "7C16 9817 7904 212D D58C  B4D1 9322 C330 474A 19E8"
+
+# Repos specific variables
+try:
```
Comment (Fdall) [inline rudder-server-relay/SOURCES/rudder-pkg-files/rudderPkgUtils.py]: We should not fail if the command used does not involve the repos, like rudder-pkg install-file or rudder-pkg list

**Comment by Fdall** [comment]
Commit modified

**Comment by Fdall** [comment]
Commit modified

**Comment by Fdall** [comment]
Commit modified

File: rudder-server-relay/SOURCES/rudder-pkg (hunk 1 above)
Comment (peckpeck) [reply rudder-server-relay/SOURCES/rudder-pkg]: Still valid

File: rudder-server-relay/SOURCES/rudder-pkg-files/rudderPkgUtils.py (hunk 2 above)
Comment (peckpeck) [reply rudder-server-relay/SOURCES/rudder-pkg-files/rudderPkgUtils.py]: still valid

File: rudder-server-relay/SOURCES/rudder-pkg-files/rudderPkgUtils.py (hunk 3 above)
Comment (peckpeck) [reply rudder-server-relay/SOURCES/rudder-pkg-files/rudderPkgUtils.py]: indeed

**Comment by Fdall** [comment]
Commit modified

**Comment by Fdall** [comment]
Moved to https://github.com/Normation/rudder/pull/2235

**Pull request closed without merging**